src/api/profile/content-types/profile/schema.json

```diff
@@ -44,6 +44,18 @@
       "config": {
         "provider": "azure-storage"
       }
+    },
+    "location": {
```

Comment:
Maybe we can give it a more specific name? Is location a zip code, country, time zone or something else?